packages/devtools/pubspec.yaml

```diff
@@ -38,11 +38,10 @@ dependencies:
 #    path: ../../third_party/packages/plotly
 
 dev_dependencies:
-  build_runner: 1.0.0
+  build_runner: ^1.0.0
   build_web_compilers: any
   matcher: ^0.12.3
   test: ^1.0.0
-  webdev: 1.0.1
```

Comment:
This dep is intentional here, to allow the workflow to be `pub run webdev serve` (i.e., using the project pub run instead of a pub global activate and a pub global run). It also ensures that we'll all using the same version of webdev (although the dart2js and ddc versions may actually be the most critical here).

If you think there's a better workflow for us here, we're happy to entertain that.

Reply:
The intended workflow of `WebDev` users is to global activate the package. We also intend for them to update the package fairly frequently as we add new features. See this issue: https://github.com/dart-lang/webdev/issues/114. This becomes a little awkward if it is a dev dependency. At the very least it should not be pinned. I'm willing to remove this and go through the documentation to update usage.

Reply:
Ok I'm sold on removing the webdev dependency. It will be nice for this package to model the best practices for using webdev and pinning to a version of webdev doesn't lock the dart2js version which is the the more crucial version to lock.